docs/installation/ubuntulinux.md

```diff
@@ -190,8 +190,11 @@ When users run Docker, they may see these messages when working with an image:
     WARNING: Your kernel does not support cgroup swap limit. WARNING: Your
     kernel does not support swap limit capabilities. Limitation discarded.
 
-To prevent these messages, enable memory and swap accounting on your system. To
-enable these on system using GNU GRUB (GNU GRand Unified Bootloader), do the
+To prevent these messages, enable memory and swap accounting on your
```

Comment:
Thanks @vincentbernat! This could use some reconstruction.  The "even when not used" is tacked on and you get better readability without the parentheticals.

---> https://gist.github.com/moxiegirl/2ecded20f4c7a28c3779

To prevent these messages, enable memory and swap accounting on your system.
Enabling memory and swap accounting does induce both a memory overhead and a
performance degradation even when Docker is not in use. The memory overhead is
about 1% of the total available memory. The performance degradation is roughly
10%. 

To enable memory and swap on system using GNU GRUB (GNU GRand Unified
Bootloader), do the following: